s3config-test.yaml

```diff
@@ -69,6 +69,7 @@ S3_SERVER_CONFIG:                                     # Section for S3 Server
    S3_REDIS_SERVER_PORT: 6379                           # In case if redis is used for kvs contains redis server port
    S3_SERVER_MOTR_ETIMEDOUT_MAX_THRESHOLD: 100          # Number of ETIMEDOUT errors per monitoring window before s3server restart
    S3_SERVER_MOTR_ETIMEDOUT_WINDOW_SEC: 2               # Monitoring window for motr ETIMEDOUT errors in seconds
+   S3_SERVER_ENABLE_MOTR_ADDB_DUMP: true                # If set to true, then addb dump will be collected
```

Comment:
it is not only motr addb - they are independent each other, so I think it is better to rename it

Reply:
done.